ipv-stub/template.yaml

```diff
@@ -101,9 +117,9 @@ Resources:
                         type: object
               x-amazon-apigateway-integration:
                 type: "aws_proxy"
-                httpMethod: "GET"
+                httpMethod: "POST"
```

Comment:
/reverification API here is implemented as GET atm

Reply:
This caught me out as well.  The integration is always done as a POST regardless of the originating method.

Reply:
Ah, TIL